tests/test_error_handlers.py

```diff
@@ -6,85 +6,58 @@
 
 def test_db_connection_error():
     """test that an exception is raised if there's an error connecting to the database"""
-    with pytest.raises(error_handlers.DatabaseConnectionError) as err:
+    with pytest.raises(error_handlers.DatabaseConnectionError):
         validators.DbValidators.connect_to_db("bad-db-uri-to-check-exception-raise")
 
-    assert str(err.value) == "500 Internal Server Error: " \
-                             "An error occurred while connecting to the database"
-
 
 def test_table_creation_error(cnxn, cursor):
     """test that an exception is raised if there's an error creating the db tables"""
     sttmt = ""
-    with pytest.raises(error_handlers.TableCreationError) as err:
+    with pytest.raises(error_handlers.TableCreationError):
         validators.DbValidators.create_tables(cnxn, cursor, sttmt)
```

Comment:
You don't have a test in this method. Why is that?

Reply:
The test is there. I'm testing that an exception is raised if the wrong data is passed in. If the exception is not raised, the test fails with a `DID NOT RAISE...` error. You can read more about it [here](https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions)